### ISSUE TITLE
Can't mount while Morph

### ISSUE THREAD

**Issue by Spriest2** [RetroWoW/RetroWoW#40]
like the title says. While morphed it says "You are in shapeshift form" and doesnt let you use your mount

**Comment by mentos987** [comment]
Same with the undead morph from noggenfogger.

**Comment by Feltyham** [comment]
With the implementation of the new core, more things went "blizz-like".

It was a bug in the first place being able to mount while shapeshifted. The new core fixed that bug.

**Comment by mentos987** [comment]
Well, it was a fun bug that added to the experience. Also, I don't wanna have to remove my noggenfogger morph all the time when I just want it in order to shrink.

**Comment by Feltyham** [comment]
Yeah I find myself spamming mount and being like "oh yeah I can't mount in skelly" 

I'm starting to break the habit but I know the feeling.
